fastlane_core.gemspec

```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'plist', '~> 3.1' # needed for parsing provisioning profiles
   spec.add_dependency 'terminal-table', '~> 1.4.5' # options summary
   spec.add_dependency 'sentry-raven', '~> 0.15' # Optional crash reporting
+  spec.add_dependency 'nokogiri', '>= 1.6.7.1' # needed for parsing pkg distribution metadata
```

Comment:
Sorry, no, we actively removed `nokogiri` as dependency, this causes to many problems, please use an alternative instead.